Matveibos/hometask-6/scrapper.rb

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'csv'
+require 'capybara'
+require 'capybara/dsl'
+
+PATH_TO_CONTAINER_SPORT = '/html/body/div[1]/div/div/div/div/div/div[6]/div[5]/div[1]'
+PATH_TO_CONTAINER_PEOPLES = '/html/body/div[1]/div/div/div/div/div/div[7]/ul'
+PATH_TO_CONTAINER_CATALOG = '/html/body/div[1]/div/div/div/div/div/div[4]/div[2]/div/div[1]'
+PATH_TO_CONTAINER_SUB_CATALOG = '/html/body/div[1]/div/div/div/div/div/div[7]/ul'
+
+# module find information
+module Finder
+  include Capybara::DSL
+
+  Capybara.default_driver = :selenium
+  Capybara.app_host = 'https://www.onliner.by'
+  Capybara.visit('/')
+
+  def find_images(path, comment)
+    @answer << ["\n#{comment}:"]
+    within(:xpath, path) do
+      page.all('img').each do |img|
+        @answer << [(img['src']).to_s]
+      end
+    end
+  end
+
+  def find_titles(path, tag, style_class, comment)
+    @answer << ["\n#{comment}:"]
+    within(:xpath, path) do
+      page.all(tag, class: style_class).each do |title|
+        @answer << [title.text.to_s]
+      end
+    end
+  end
+
+  def find_symbols(path, tag, style_class, comment)
+    @answer << ["\n\n#{comment}:"]
+    within(:xpath, path) do
+      page.all(tag, class: style_class).each do |title|
+        @answer << [title.text[0...200].to_s]
+      end
+    end
+  end
+end
+
+# Write to array
+class Answer
+  include Finder
+
+  def initialize
+    @answer = []
+  end
+
+  attr_reader :answer
+end
+
+# Write to file
+class FileWriter
+  def write(answer)
+    CSV.open('file.csv', 'wb') do |csv|
+      answer.each do |row|
+        csv << row
+      end
+    end
+  end
+end
+
+infomation = Answer.new
+csv_file = FileWriter.new
+
+infomation.find_images(PATH_TO_CONTAINER_SPORT, 'sport image')
```

Comment:
Let's define an array of configuration which will contain path, tag, class and comment and then just traverse through this array here, rather than write everything line by line